udata_front/theme/gouvfr/assets/js/components/search/OrganizationSearch/OrganizationSearchOption.vue

```diff
@@ -1,27 +1,21 @@
 <template>
-  <div class="fr-grid-row fr-grid-row--gutters fr-grid-row--center border-default-grey border-bottom">
+  <div class="fr-grid-row fr-grid-row--center fr-py-1w fr-enlarge-link border-default-grey border-bottom">
     <!-- Logo/Placeholder column -->
     <div class="fr-col-auto fr-display-flex justify-center fr-my-auto">
       <img v-if="logo" :src="logo" alt="Organization Logo" height="30" class="fr-mx-1w" />
       <Placeholder v-else type="organization" :size="30" class="fr-mx-1w" />
     </div>
     <!-- Name column -->
-    <div class="fr-col fr-display-flex align-items-center fr-my-auto">
-      <span class="fr-name">{{ name }}</span>
-    </div>
-    <!-- Link column -->
-    <div class="fr-col-auto fr-display-flex align-items-center justify-end fr-my-auto fr-mr-1w">
-      <a target="_blank" :href="link">{{ t('See Organization') }}</a>
-    </div>
+    <a class="fr-col fr-display-flex align-items-center justify-center fr-my-auto" :href="link">
```

Comment:
```suggestion
    <a class="fr-col" :href="link">
```

all these are unnecessary